ci/jenkins/jobs/macros.yaml

```diff
@@ -232,9 +232,10 @@
           make clean
           docker pull antrea/openvswitch --all-tags
           docker images | grep 'antrea-ubuntu' | awk '{print $3}' | xargs -r docker rmi || true
-          # Clean up dangling images generated in previous builds. Recent ones must be excluded
-          # because they might be being used in other builds running simultaneously.
+          # Clean up dangling and unused antrea-ubuntu images generated in previous builds.
+          # Recent ones must be excluded because they might be being used in other builds running simultaneously.
           docker image prune -f --filter "until=1h" || true
+          docker image prune -f -a --filter "until=1h" --filter "label=type=antrea-ubuntu" || true
```

Comment:
Doesn't L234 do it?

Reply:
Thanks for pointing out. Actually, L234 does not fit any more:
* All jobs start on the bootstrap node. Images of other jobs should not be affected.
* Jobs of the same commit share one image id. Such error will be returned: "image is referenced in multiple repositories".